bundles/org.eclipse.ui.workbench/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.ui.workbench; singleton:=true
-Bundle-Version: 3.125.100.qualifier
+Bundle-Version: 3.126.0.qualifier
```

Comment:
TODO: Once confirmed, also update versions in MANIFEST.MF of depending projects.

Reply:
Should now be 3.127.0 as 3.126 was released in the meantime.